resources/js/features/home/components/+sidebar/AchievementOfTheWeek/AotwUnlockedIndicator/AotwUnlockedIndicator.tsx

```diff
@@ -0,0 +1,20 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { usePageProps } from '@/common/hooks/usePageProps';
+
+export const AotwUnlockedIndicator: FC = () => {
+  const { achievementOfTheWeek } = usePageProps<App.Http.Data.HomePageProps>();
+
+  const { t } = useTranslation();
+
+  if (!achievementOfTheWeek?.doesUserHaveUnlock) {
+    return null;
+  }
+
+  return (
+    <div data-testid="aotw-progress" className="bg-neutral-950 px-2 py-1.5">
+      <p className="text-center text-xs text-neutral-300">{t('Unlocked.')}</p>
```

Comment:
I find the period here out of place. It's not a sentence.

Reply:
No problem, went ahead and removed the period.